integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageOverride.java

```diff
@@ -0,0 +1,405 @@
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Level;
+
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.LoggerHelper;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.Operator.RestCertType;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * This JUnit test is used for testing
+ * Wdt Config Override with Model File(s) to existing MII domain
+ *
+ * <p>This test is used for creating domain using model in image.
+ */
+
+public class ItModelInImageOverride extends MiiBaseTest {
+  private static Operator operator;
+  private static Domain domain;
+  private static String domainNS;
+  private static String testClassName;
+  private static StringBuffer namespaceList;
+  private static final String configMapSuffix = "-mii-config-map";
+  private static final String dsName = "MyDataSource";
+  private static final String readTimeout_1 = "30001";
+  private static final String readTimeout_2 = "30002";
+
+  /**
+   * This method does the initialization of the integration test
+   * properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception exception if initializing the application properties
+   *          and creates directories for results fails.
+   */
+  @BeforeAll
+  public static void staticPrepare() throws Exception {
+    namespaceList = new StringBuffer();
+    testClassName = new Object() {
+    }.getClass().getEnclosingClass().getSimpleName();
+    // initialize test properties and create the directories
+    initialize(APP_PROPS_FILE, testClassName);
+  }
+
+  /**
+   * This method creates the result/pv root directories for the test.
+   * Creates the operator if its not running.
+   *
+   * @throws Exception exception if result/pv/operator/domain creation fails
+   */
+  @BeforeEach
+  public void prepare() throws Exception {
+    createResultAndPvDirs(testClassName);
+
+    // create operator1
+    if (operator == null) {
+      Map<String, Object> operatorMap = createOperatorMap(getNewSuffixCount(),
+          true, testClassName);
+      operator = TestUtils.createOperator(operatorMap, RestCertType.SELF_SIGNED);
+      Assertions.assertNotNull(operator);
+      domainNS = ((ArrayList<String>) operatorMap.get("domainNamespaces")).get(0);
+      namespaceList.append((String)operatorMap.get("namespace"));
+      namespaceList.append(" ").append(domainNS);
+    }
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories.
+   *
+   * @throws Exception exception
```

Comment:
fix this

Reply:
stopping here - resubmit when you fix the basic stuff..

Reply:
fixed all :(

Reply:
fixed all :(